### ISSUE TITLE
Please add "READ-only" option from command line.

### ISSUE THREAD

**Issue by achangs** [coteditor/CotEditor#1099]
I love using cot editor, especially the command line integration.
Often, I want to open a file in read-only mode, so that I don't risk modifying a file unintentionally.

Something like this:
cot -r <filename>

where the "-r" option would invoke the read-only option.

Thank you!!

**Comment by codifyer** [comment]
Yes, that would be a really useful. Wonder why most editors don't have this essential feature. But it would be a good one for cot.

**Comment by tibor-leo-safar-accedo** [comment]
One more vote for the read-only mode! Thanks!

**Comment by 1024jp** [comment]
The implementation has just been done!
The feature will be delivered with CotEditor 5.1.0, which will be released in early spring.
